hazelcast-jet-s3/src/main/java/com/hazelcast/jet/s3/S3Sinks.java

```diff
@@ -0,0 +1,153 @@
+/*
+ * Copyright (c) 2008-2019, Hazelcast, Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hazelcast.jet.s3;
+
+import com.amazonaws.regions.Regions;
+import com.amazonaws.services.s3.AmazonS3;
+import com.hazelcast.jet.function.FunctionEx;
+import com.hazelcast.jet.function.SupplierEx;
+import com.hazelcast.jet.pipeline.Sink;
+import com.hazelcast.jet.pipeline.SinkBuilder;
+
+import javax.annotation.Nonnull;
+
+/**
+ * Contains factory methods for creating AWS S3 sinks.
+ */
+public final class S3Sinks {
+
+    private static final int DEFAULT_BATCH_SIZE = 1024;
```

Comment:
I think we need higher default. This will create many very small files.

Reply:
What do you have in mind ?